_posts/2023-04-16-18f-checks-in-with-the-dawson-project-at-the-us-tax-court.md

```diff
@@ -19,9 +18,13 @@ excerpt: >
   The United States Tax Court is an independent federal court that settles disputes between taxpayers and the Internal Revenue Service. Since the 1980s, the Court has used a modified commercial software product to manage its operations. Last year the Court introduced DAWSON, a modern open-source case management system developed with assistance from an industry partner and 18F. DAWSON streamlined Court operations and changed the way the public interacts with the Court.
 ---
 
+_This is the third in a series of blog posts where 18F checks in with our partners throughout the government. Previously in this series: [18F Checks In With Emily Read and the USGS Water Resources Mission Area Projects](https://18f.gsa.gov/2023/02/24/18f-checks-in-with-emily-read-and-the-usgs-water-resources-mission-area-projects/)._
```

Comment:
I noticed that the last check-in blog post referenced another one, so I decided to imitate that. Let me know if we don't want this, or suggest edits if you'd like!

Reply:
@julialeague Love it! 🙌